bin/cdist

```diff
@@ -25,6 +26,8 @@ import logging
 import os
 import sys
 
+LOG = logging.getLogger("cdist")
```

Comment:
```
$ grep -rFI getLogger
flock.py:log = logging.getLogger('cdist-flock')
scan/commandline.py:log = logging.getLogger("scan")
scan/scan.py:log = logging.getLogger("scan")
emulator.py:        self.log = logging.getLogger(self.target_host[0])
inventory.py:        self.log = logging.getLogger("inventory")
inventory.py:        log = logging.getLogger("inventory")
log_server.py:        logger = logging.getLogger(record.name)
log_server.py:def setupLogServer(socket_dir, log=logging.getLogger(__name__)):
argparse.py:    logging.getLogger().setLevel(_verbosity_level[args.verbose])
argparse.py:    log = logging.getLogger("cdist")
util/ipaddr.py:    log = logging.getLogger(host)
util/ipaddr.py:    log = logging.getLogger(host)
sphinxext/manpage.py:logger = logging.getLogger(__name__)
preos/debootstrap/debootstrap.py:        log = logging.getLogger(cls.__name__)
mputil.py:log = logging.getLogger("cdist-mputil")
exec/remote.py:        self.log = logging.getLogger(self.target_host[0])
exec/local.py:        self.log = logging.getLogger(self.target_host[0])
preos.py:log = logging.getLogger("PreOS")
log.py:    del logging.getLogger().handlers[:]
log.py:    del logging.getLogger().handlers[:]
log.py:    del logging.getLogger().handlers[:]
log.py:    del logging.getLogger().handlers[:]
core/cdist_type.py:    log = logging.getLogger("cdist-type")
core/explorer.py:        self.log = logging.getLogger(self.target_host[0])
core/manifest.py:        self.log = logging.getLogger(self.target_host[0])
test/manifest/__init__.py:        self.log = logging.getLogger(self.target_host[0])
message.py:log = logging.getLogger(__name__)
shell.py:log = logging.getLogger(__name__)
config.py:        log = logging.getLogger("config")
config.py:        log = logging.getLogger(host)
config.py:        self.log = logging.getLogger(self.local.target_host[0])
banner.py:log = logging.getLogger(__name__)
```

do we want to follow same behavior or go with UPPERCASE for file-scope global stuff?

Reply:
I think going with lower-case `log` is just fine if it has been used consistently across the code base.